pkg/registration/spoke/spokeagent.go

```diff
@@ -418,6 +459,54 @@ func (o *SpokeAgentConfig) RunSpokeAgentWithSpokeInformers(ctx context.Context,
 		)
 	}
 
+	var hubAcceptController, hubTimeoutController factory.Controller
+	hubKubeConfigSecretAutoGenCtx, cancelHubKubeConfgiSecretAutoGen := context.WithCancel(ctx)
+	defer cancelHubKubeConfgiSecretAutoGen()
+	if features.SpokeMutableFeatureGate.Enabled(ocmfeature.MultipleHubs) {
+		hubAcceptController = registration.NewHubAcceptController(
+			o.agentOptions.SpokeClusterName,
+			hubClusterInformerFactory.Cluster().V1().ManagedClusters(),
+			func(ctx context.Context) error {
```

Comment:
this is duplicate with line 492, should build the same one.

Reply:
Agree, merged.